tests/foreman/ui/test_discoveredhost.py

```diff
@@ -141,6 +135,234 @@ def setUpClass(cls):
         cls.discovery_auto.value = 'True'
         cls.discovery_auto.update({'value'})
 
+        # Create a new Life-Cycle environment
```

Comment:
Sure @oshtaier. I'll file an issue to handle this change in another PR, if you'r fine with it.  For now, I'll share the test results